### DISCUSSION TITLE
Importing sqlite map files

### DISCUSSION THREAD

**Discussion by barteis** [osmandapp/OsmAnd#15589] [Q&A] [answered]
What's the best way to import sqlite map files for use in OsmAnd? I used to use OsmAnd/tiles, but it's no longer available.  

And what about importing them in OsmAnd iOS?  

Thanks!

**Comment by vshcherb** [comment, marked as answer]
- Android. If you just download click on *.sqlite it will suggest to import using OsmAnd
- iOS. Same here if you sync it with Apple Cloud and open with OsmAnd.

**Reply by barteis** [reply]
Thank you both.

**Comment by mariush444** [comment]
till android 11.0 you can move/create osmand directory in Android/media instead Android/data then you have access to the folders as previously
